user_docs/en/userGuide.t2t

```diff
@@ -2593,52 +2593,67 @@ For more information, please see the [NVDA Developer Guide https://www.nvaccess.
+Add-ons have associated actions that you can access through an [actions menu #AddonStoreActions], such as install, help, disable, and remove.
+Available actions will change based on whether the add-on is installed or not, and whether it is enabled or disabled.
 
 ++++ Add-on list views ++++[AddonStoreFilterStatus]
+There are different views for installed, updatable, available and incompatible ad-ons.
 To change the view of add-ons, change the active tab of the add-ons list using ``ctrl+tab``.
-There are different views for installed, updatable, available and incompatible.
+You can also ``tab`` to the list of views, and move through them with the ``leftArrow`` and ``rightArrow`` keys.
+
+++++ Filtering for enabled or disabled add-ons ++++[AddonStoreFilterEnabled]
+Normally, an installed add-on is "enabled", meaning that it is running and available within NVDA.
+However, some of your installed add-ons may be set to the "disabled" state.
+This means that they will not be used, and their functions won't be available during your current NVDA session.
+You may have disabled an add-on because it conflicted with another add-on, or with a certain application.
+NVDA may also disable certain add-ons, if they are found to be incompatible during an NVDA upgrade; though you will be warned if this is going to happen.
+Add-ons can also be disabled if you simply don't need them for a prolonged period, but don't want to uninstall them because you expect to want them again in the future.
 
-++++ Include incompatible add-ons ++++[AddonStoreFilterEnabled]
-Installed and incompatible add-ons can be filtered by enabled or disabled.
+The lists of installed and incompatible add-ons can be filtered by their enabled or disabled state.
 The default shows both enabled and disabled add-ons.
 
 ++++ Include incompatible add-ons ++++[AddonStoreFilterIncompatible]
-Available and updatable add-ons can be filtered to include [incompatible add-ons #incompatibleAddonsManager] available to install.
+Available and updatable add-ons can be filtered to include [incompatible add-ons #incompatibleAddonsManager] that are available to install.
 
 ++++ Filter add-ons by channel ++++[AddonStoreFilterChannel]
-To list add-ons only for specific channels, change the "Channel" filter selection.
-
 Add-ons can be distributed through up to four channels:
 - Stable: The developer has released this as a tested add-on with a released version of NVDA.
-- Beta: This add-on may need further testing.
+- Beta: This add-on may need further testing, but is released for user feedback.
 Suggested for early adopters.
 - Dev: This channel is suggested to be used by add-on developers to test unreleased API changes.
 NVDA alpha testers may need to use a "Dev" version of their add-ons.
 - External: Add-ons installed from external sources, outside of the add-on store.
 -
 
-++++ Search add-ons ++++[AddonStoreFilterSearch]
-Add-ons can be filtered by display name, publisher and description.

```

Comment:
could you please re-include the list of strings which are used in the search filter

Reply:
I didn't know we were going to have that working by release. Will revert.

Reply:
That's the current search behaviour. We have a proposal to use n-gram or similar searches to rank results by match.